### PR TITLE
Add visual regression testing

### PR DESCRIPTION
**Pull request by Br3nda**
Percy takes screen shots of capybara tests, and uploads them -- then compares them between branches on pull requests.

This will help us catch visual regressions.

Percy is one of the checks on the pull request.

### CODE REVIEW

**Comment by Br3nda** [comment]
We'll also need to add our users to percy to approve the changes.

**Comment by djones** [comment]
Awesome stuff! 💜 Love that you made your Percy project publicly viewable too. Let us know if you need a hand with anything @Br3nda.

File: .travis.yml [hunk 1]
```diff
@@ -20,6 +21,8 @@ addons:
     - google-chrome-stable
 script:
   - brakeman --run-all-checks --exit-on-warn --format plain .
-  - bundle exec rspec spec
+  - npx percy exec -- bundle exec rspec spec
 after_script:
   - ./cc-test-reporter after-build --exit-code $TRAVIS_TEST_RESULT
+PERCY_TOKEN:
```
Comment (eoinkelly) [inline .travis.yml:27]: @Br3nda Should this key be in the Travis env instead of hard-coded here?

**Review by eoinkelly — CHANGES_REQUESTED** [review]
LGTM except possibly for the PERCY_TOKEN thing (see other comment)

File: .travis.yml (hunk 1 above)
Comment (Br3nda) [reply .travis.yml:27]: If it's in Travis, it isn't available in PRs from forks.

**Review by eoinkelly — APPROVED** [review]

**Pull request merged**